keras_cv/losses/iou_loss_test.py

```diff
@@ -57,23 +57,3 @@ def test_output_shape_relative(self):
         iou_loss = IoULoss(bounding_box_format="rel_xyxy")
 
         self.assertAllEqual(iou_loss(y_true, y_pred).shape, ())
```

Comment:
please add numerical tests here. It should have some easy to understand numerics and comments like this:
https://github.com/keras-team/keras-cv/blob/master/keras_cv/losses/penalty_reduced_focal_loss_test.py#L53-L91